functionnal_tests/cypress/integration/publication/toggle_comments.js

```diff
@@ -0,0 +1,30 @@
+import { PAGES } from '../../support/urls_commands'
+
+const publishButton = '.publications__publishArea__buttons__submit'
+const text = 'Hello, world'
+
+describe('Publications page', () => {
+  beforeEach(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then((workspace) => {
+      cy.visitPage({
+        pageName: PAGES.PUBLICATION,
+        params: { workspaceId: workspace.workspace_id },
+        waitForTlm: true
+      })
+    })
+  })
+
+  it('should have translations', () => {
+    cy.get('#wysiwygTimelineCommentPublication').type(text)
```

Comment:
```suggestion
  it("should change button's label according to its state", () => {
    cy.get('#wysiwygTimelineCommentPublication').type(text)
```

I think it's a good idea to set the language to french in the beginning with `cy.changeLanguage` to be sure that the translations work.
Also, you can add a comment to your publication and test the button in theis case

Reply:
Oops, thanks. 
Good idea, I'll add this.